spec/knapsack_pro/config/temp_files_spec.rb

```diff
@@ -0,0 +1,22 @@
+describe KnapsackPro::Config::TempFiles, :clear_tmp do
+  describe '.temp_directory_path' do
+    let(:gitignore_file_path) { '.knapsack_pro/.gitignore' }
+
+    subject { described_class.temp_directory_path }
+
+    it 'returns temporary directory path' do
+      expect(subject).to eq '.knapsack_pro'
+    end
+
+    it 'creates .gitignore file' do
+      expect(File.exists?(gitignore_file_path)).to be false
+      subject
+      expect(File.exists?(gitignore_file_path)).to be true
+    end
+
+    it '.gitignore file has content' do
```

Comment:
Do I understand correctly that the most important line in the file's content is the one with the asterisk (`*`)? If so, then maybe it'd be better to check the inclusion of it instead?